### ISSUE TITLE
[CHORE] Clean-up string parsing from templates to Typescript

### ISSUE THREAD

**Issue by TiBiBa** [Felienne/hedy#2271]
Currently we perform quite some string parsing from the HTML templates to the Typescript code. For example, to return the user a message that they haven't enter a name we get this string in the template though Jinja and then add it as an argument to a Typescript function. 

This is cumbersome and likely to introduce bugs (such as quotes in the strings). We can escape all these errors by surrounding them with `|tojson` parsing but this does not make any sense. We should dependent on the server to return specific messages, not on our own "houtje-touwtje" string parsing. This is introduced due to merging #2210 and losing access to all translation in the Typescript and should be cleaned-up now.

First things that come to mind that need clean-up:

- Creation of a class
- Renaming of a class
- Inviting a student
- Change password student
- Creating an adventure
- Basically anything that uses the `modal.prompt` and verifies for an empty string

When we want to keep the string we should always enclose it with `|tojson` to prevent quotes and other special characters from escaping the function.

**Comment by TiBiBa** [comment]
On second thoughts we can add these string to the function calls as long as they are always used. For the "value is empty" response we should still depend on the server.
